pages/instructorQuestions/instructorQuestions.ejs

```diff
@@ -30,6 +30,11 @@
               <% questions.forEach(function(question) { %>
               <tr>
                 <td class="align-middle">
+                  <% if (question.sync_errors) { %>
+                    <i class="fa fa-times text-danger" aria-hidden="true"></i>
+                  <% } else if (question.sync_warnings) { %>
+                    <i class="fa fa-exclamation-triangle text-warning" aria-hidden="true"></i>
+                  <% } %>
```

Comment:
Can we make these link to the `edit` page for the corresponding `info.json` file, or have a popover containing a message about the error and link to `edit`, or something like that? I guess I’m thinking that we want users to see the error, and be able to click it to figure out what it means and how to fix it. A similar current pattern is the issues counter on questions that can be clicked to go straight to the issues list: https://github.com/PrairieLearn/PrairieLearn/blob/master/pages/instructorQuestions/instructorQuestions.ejs#L34

Reply:
I'm kind of torn on this - simply clicking on the question will show that information, and said information will probably be a lot more legible in a full-width browser window than in a tooltip. I'm not sure if much is gained by putting it inline on the question/assessment page.

Reply:
You are right, popovers would suck for this 😄. I do still think that the `edit` link (plus showing errors there) would be awesome.